src/style.css

```diff
@@ -1 +1,33 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star:last-child {
+  margin-bottom: 0;
+}
```

Comment:
![image](https://github.com/mate-academy/layout_stars/assets/96409563/35f331d2-4a19-4ba4-a4c4-618bcace24d4)

Reply:
You should not add margins to block